pliers/filters/video.py

```diff
@@ -1,17 +1,26 @@
 ''' Filters that operate on TextStim inputs. '''
 
-from pliers.stimuli.video import VideoFrameCollectionStim
+from pliers.stimuli.video import VideoStim, VideoFrameCollectionStim
 from .base import Filter, TemporalTrimmingFilter
 
+import numpy as np
+
 
 class VideoFilter(Filter):
 
     ''' Base class for all VideoFilters. '''
 
+    _input_type = VideoStim
+
+
+class VideoFrameFilter(Filter):
```

Comment:
I think we can probably drop this if the only thing it's used for is to set the `_input_type`... that could just be done in each subclass. Unless you think it will make sense to add functionality here down the line, or there's somewhere else it's important.